### ISSUE TITLE
Group CloudVars broken post-migration

### ISSUE THREAD

**Issue by 3x1t-5tyl3** [Yellow-Dog-Man/Resonite-Issues#3133]
### Describe the bug?

Currently all of my CloudVariables from Neos that are on my group (such as my headless cloudvars and some tools I made) are now broken in it's entirety.

### To Reproduce

Not sure to be honest. They just don't work for me at all. And making new ones seems rather asanine and hard to keep track of.
As-well as the fact Group CV's are broken further #537 

Sadly I have no way of reproducing this.

### Expected behavior

The cloudvar's to work as they used to.

### Resonite Version Number

Beta 2024.10.23.15

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Vive Pro Eye/Desktop

### Log Files

N/A; Sadly i have no way to reproduce this and I couldn't find anything in the logs when i was migrating. Though I do know that migration showed zero errors.

### Additional Context

Not quite sure what's happening.
I explicitly went for a Patreon Sub so I can get my cloudvars back thinking "Oh well; I guess I need to pay to use the ones I made in neos"
A week later they're still broken to hell and back for me and my account.

Though User-variables work at-least.

### Reporters

3x1t_5tyl3

**Comment by shiftyscales** [comment]
This sounds almost certainly to be related to #3101. I'd recommend reading through and answering the questions Frooxius had on that issue / providing your information as requested here, e.g. U-ids, etc.

Do you also have any issues with cloud vars in general that aren't group-based, @3x1t-5tyl3?

**Comment by 3x1t-5tyl3** [comment]
> This sounds almost certainly to be related to [#3101](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/3101). I'd recommend reading through and answering the questions Frooxius had on that issue / providing your information as requested here, e.g. U-ids, etc.
> 
> Do you also have any issues with cloud vars in general that aren't group-based, [@3x1t-5tyl3](https://github.com/3x1t-5tyl3)?

No. My user variables are all retained and kept their data too.
It's only my group variables which are unusable.

**Comment by shiftyscales** [comment]
When creating your Resonite account, had you kept the original U-id of the account from the compatible infrastructure you migrated from? Or did you reset it, @3x1t-5tyl3?

Likewise- had you migrated immediately at the time of your account creation? Or had you created a separate Resonite account, then migrated data afterwards at a later point?

**Comment by 3x1t-5tyl3** [comment]
> When creating your Resonite account, had you kept the original U-id of the account from the compatible infrastructure you migrated from? Or did you reset it, [@3x1t-5tyl3](https://github.com/3x1t-5tyl3)?
> 
> Likewise- had you migrated immediately at the time of your account creation? Or had you created a separate Resonite account, then migrated data afterwards at a later point?

I kept the original ID. And made sure to select "All" when doing migration.
I did not make a seperate account (apart from my headless account ofc)

But no. My userid is still the same as in neos. The group id is still the same as it was in neos.
U-3x1t-5tyl3 and G-Exitian
